### ISSUE TITLE
[Feature Request] Small Layout Tweak to optimize use of screen space

### ISSUE THREAD

**Issue by xekon** [ampache/ampache#2797]
## Describe what you don't like about Ampache, or what it is missing

I would like to move the heart icon to the same line as the ratings while browsing albums, artists, etc
The reason for this request is to just make a little bit better use of the screen space.
Here is an example I did in Gimp:

![2021-03-04_16-26](https://user-images.githubusercontent.com/106644/110049497-cdbe2b00-7d06-11eb-80bd-0e2863d7c422.png)

## Describe how you would like the feature to work**

This could be the new default, or there could be a setting for compressed layout or something to that affect.

**Comment by xekon** [comment]
Today I found a workaround I had not considered, just disable it.

```
nano /var/www/ampache/config/ampache.cfg.php
userflags = "false"
```

**Comment by lachlan-00** [comment]
it's an annoying item because they're separated but shouldn't really be in the grid view.

Last time i looked at it i just gave up and centered it

**Comment by xekon** [comment]
LOL! I tried looking at the code to see if I could put them both into the same cell. but it was difficult for me to figure it out but by looking at the code, however I seen something about "optional" so I then looked at the options and seen that I could just disable the feature completely. I dont really use the heart icon, only the ratings. My wife used the heart icon but said she could live without it if I need to disable it.
